apps/src/aichat/views/ModelCustomizationWorkspace.tsx

```diff
@@ -20,7 +25,9 @@ const ModelCustomizationWorkspace: React.FunctionComponent = () => {
   );
 
   const showSetupCustomization =
-    isVisible(temperature) || isVisible(systemPrompt);
+    isVisible(temperature) ||
+    isVisible(systemPrompt) ||
+    isVisible(selectedModelId);
```

Comment:
Noticed we were hiding the Setup panel if temperature and system prompt were hidden, but selected model ID was still editable. This makes sure we'll still show the Setup panel if we can change the model ID (unless this was intentinonal?)

Reply:
Not intentional. Just an oversight. Thanks!